synapse/models/inet.py

```diff
@@ -234,6 +237,51 @@ def _normPyStr(self, valu):
         except Exception as e:
             raise s_exc.BadTypeValu(valu)
 
+class Rfc2822Addr(s_types.Type):
```

Comment:
Maybe s_types.Str so you get a bunch of the below for free?

Reply:
What would we get for free?

Reply:
Go ahead and leave it as-is for now, but plz add an indxcmpr for ^= to do prefix matching :)

Reply:
sure

Reply:
added